provisioning_templates/snippet/saltstack_setup.erb

```diff
@@ -21,6 +21,8 @@ if [ -f /usr/bin/dnf ]; then
 else
   yum -t -y install salt-minion
 fi
+<% elsif @host.operatingsystem.family == 'Suse' -%>
+  /usr/bin/zypper -n install salt-minion
```

Comment:
I don't have much experience with zypper so I'll ask, doesn't this need something like `-y` or `--no-confirm`?

Reply:
--non-interactive, -n	Do not ask anything, use default answers
				automatically.

Reply:
```
           -n, --name
               Select packages by their name, don’t try to select by capabilities.
```

From https://en.opensuse.org/SDB:Zypper_manual_%28plain%29

Reply:
Global Options vs. Command Options
Syntax is zypper <global-options> <command> <command-options>

The global option -n is --non-interactive, so @sbernhard is right
BTW: -y would work too, its an alias ;)

Reply:
just because I copied it from the puppet snippet :)